tasks/service_ctl_tomcat.yml

```diff
@@ -0,0 +1,17 @@
+---
+# tasks file for universal-tomcat - start tomcat with startup.sh script
+
+- name: Install Service
+  template:
+    src: tomcat.conf.j2
+    dest: /etc/init/tomcat.conf
+    mode: 0644
+
+- name: Reload Services To Include Tomcat
+  command: initctl reload-configuration
```

Comment:
Command Module is Non-Idempotent by default. Definitely needs a `when` clause when the service installation changed. Also does this do the same thing as the service module with `state:reloaded`